source/nielvisiii/academicIO.py

```diff
@@ -233,6 +234,9 @@ def __write_to_cnfg_register(bank, cnfg):
             while not(cnfg == self.cnfg[bank].read() and self.ready[bank].read()):
                 pass
 
+            # wait 500 us to apply the AI registers once configuration is modified. 12*1000/40 M = 300 us < 500 us
+            time.sleep(0.5)
```

Comment:
If we does not wait here, sometimes the readback value is wrong in the following stimulation:
write -> read -> write -> read ...